tests/unit/os_test.ts

```diff
@@ -79,7 +79,9 @@ Deno.test(
     ) => {
       const src = `
       console.log(
-        ${JSON.stringify(Object.keys(expectedEnv))}.map(k => Deno.env.get(k))
+        ${
+        JSON.stringify(Object.keys(expectedEnv))
+      }.map(k => Deno.env.get(k) ?? null)
```

Comment:
why is this change required? I assume it's unrelated?

Reply:
`undefined` isn't valid JSON, but `null` is (it does a `JSON.parse` below). I ran into this issue while developing where I introduced a bug, then it was `undefined`, then the error message was not great.